arelle/plugin/validate/HMRC/__init__.py

```diff
@@ -43,12 +43,19 @@
     "charities": commonMandatoryItems | {
         "DateAuthorisationFinancialStatementsForIssue", "DirectorSigningFinancialStatements",
         "EntityDormantTruefalse", "EntityTradingStatus",
-        "AccountingStandardsApplied", "AccountingStandardsApplied"},
+        "AccountingStandardsApplied", "AccountsStatusAuditedOrUnaudited"},
```

Comment:
Was this always wrong, or does this change only apply to the newer taxonomy?  Should we instead check `val.modelXbrl.nameConcepts.get("AccountsStatusAuditedOrUnaudited", ())` and add a `charities-2023` key instead?

Reply:
This seems to have always been wrong.  I can't find a version of the Joint Filing Common Validation Checks prior to v4. In V4 both  "AccountingStandardsApplied" and "AccountsStatusAuditedOrUnaudited" have required for the charity taxonomies from 2015 to present.